components/core/DownloadManager.py

```diff
@@ -7,349 +7,437 @@
 verbose = False
```

Comment:
Are we using tabs to format stuff in Bassa? 🤔It's spaces in other files I think 🤯

Reply:
True @JaDogg, according to Pep8 standards it is good to go with spaces rather tabs, it is good to work on API code base to meet the standards.